src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -126,19 +126,147 @@ class PerformanceLattice(BaseModel):
+            spin-echo dynamical decoupling protocol. This measurement isolates the effects of incoherent processes (measured in s)
+        T2EchoEnsemble (Decimal): Dephasing time of an ensemble of non-interacting qubits distributed over the user region, in the absense
+            of drive, as measured by a spin-echo dynamical decoupling protocol. This measurement isolates the effects
+            of incoherent processes. (measured in s)
+        T2RabiSingle (Decimal): Typical decoherence time of a single driven qubit, as measured by a Rabi oscillation protocol with variable
+            pulse duration a maximum Rabi frequency. Includes coherent and incoherent processes.(measured in s)
+        T2RabiEnsemble (Decimal): Decoherence time of an ensemble of non-interacting driven qubits distributed over the user region, as measured
+            by a Rabi oscillation protocol with variable pulse duration at maximum Rabi frequency. Includes
+            coherent and incoherent processes. (measured in s)
+        T2BlockadedRabiSingle (Decimal): Typical decoherence time of a single pair of driven blockaded qubits, as measured by a Rabi oscillation
+            protocol with variable pulse duration a maximum Rabi frequency. Includes coherent and incoherent processes. (measured in s)
+        T2BlockadedRabiEnsemble (Decimal): Decoherence time of an ensemble of pairs of driven blockaded qubits distributed over the user region
+            (different pairs do not interact with each other), as measured by a Rabi oscillation protocol with variable pulse duration at
+            maximum Rabi frequency. Includes coherent and incoherent processes. (measured in s)
+        detuningError (Decimal): Systematic error from specified value of the global detuning averaged over the user
+            region. (measured in rad/s)
+        detuningInhomogeneity (Decimal): RMS inhomogeneity of the detuning over the user region. (measured in rad/s)
+        rabiAmplitudeRampCorrection (List[RabiCorrection]): dynamic correction curve of effective single-qubit
+            on-resonant Rabi oscillation frequency driven by a triangular amplitude waveform, relative
     """
 
     rabiFrequencyErrorRel: Decimal
+    rabiFrequencyGlobalErrorRel: Decimal
+    rabiFrequencyInhomogeneityRel: Decimal
+    groundDetectionError: Decimal
+    rydbergDetectionError: Decimal
+    groundPrepError: Decimal
+    rydbergPrepErrorBest: Decimal
+    rydbergPrepErrorWorst: Decimal
+    T1Single: Decimal
+    T1Ensemble: Decimal
+    T2StarSingle: Decimal
+    T2StarEnsemble: Decimal
+    T2EchoSingle: Decimal
+    T2EchoEnsemble: Decimal
+    T2RabiSingle: Decimal
+    T2RabiEnsemble: Decimal
+    T2BlockadedRabiSingle: Decimal
+    T2BlockadedRabiEnsemble: Decimal
+    detuningError: Decimal
+    detuningInhomogeneity: Decimal
+    rabiAmplitudeRampCorrection: List[RabiCorrection]
```

Comment:
Looks like we don't define List from typing. Do we need to add from typing import List here?

Reply:
We should also consider the built in list type

Reply:
Good catch, addressed